app/src/androidTest/java/mozilla/lockbox/screenshots/ScreenshotsTest.kt

```diff
@@ -88,24 +88,35 @@ open class ScreenshotsTest {
     }
 
     @Test
-    fun testAppSettingsMenu() {
+    fun testSettingsAccountMenu() {
         Screengrab.setDefaultScreenshotStrategy(UiAutomatorScreenshotStrategy())
-        navigator.gotoSettings()
-        Screengrab.screenshot("settings-menu-screen")
-
-        navigator.gotoAccountSetting()
-        Screengrab.screenshot("settings-account-screen")
-
+        navigator.checkAtItemList()
```

Comment:
This could probably be replaced with by adding an `exists()` call in the `itemList` block below.